docs/docs/responses.mdx

```diff
@@ -97,98 +89,63 @@ to achieve the same result:
   - text: "Hey, {name}. How is your day going?"
 ```
-
-When the model predicts `utter_greet`, Rasa will randomly pick one of them.
+In this example, when `utter_greet` gets predicted as the next action, Rasa will randomly pick one 
+of the two response variations and use it.
 
 
 ### Channel-Specific Response Variations
 
-If you want to specify different response variations that depend on which channel
-the user is connected to, you can do this with channel-specific response variations.
-
-Each time your bot looks for a response variation under a given response name, it will first check
-to see if there are any channel-specific response variations for the connected channel.
-If there are, it will choose a response from **only** these response variations.
-
-If no channel-specific response variations are
-found, the bot will choose from any response variations that do not have a defined `channel`.
-
-:::caution
-It is good practice to always have at least one response variation for each
-response that has no `channel` specified. This allows your bot to respond in all
-environments, including in new environments, in the shell, and in interactive learning.
-:::
+To specify different response variations depending on which channel
+the user is connected to, use channel-specific response variations.
 
-#### Defining Channel-Specific Response Variations
+In the following example, the `channel` key makes the first response variation channel-specific for 
+the `slack` channel while the second variation is not channel-specific:
 
-Channel-specific responses are defined in a response with the `channel` key.
-
-The following example shows how you might use channel-specific responses to send a
-specific `utter_ask_game` response to the `slack` channel:
-
-```yaml-rasa title="domain.yml"
+```yaml-rasa {4} title="domain.yml"
 responses:
   utter_ask_game:
-  - text: "Which game would you like to play?"
+  - text: "Which game would you like to play on Slack?"
     channel: "slack"
-    custom:
-        blocks:
-        - type: actions
-          elements:
-          - type: button
-            text:
-              type: plain_text
-              emoji: true
-              text: "Chess :crown:"
-            value: '/inform{"game": "chess"}'
-          - type: button
-            text:
-              type: plain_text
-              emoji: true
-              text: "Checkers :checkered_flag:"
-            value: '/inform{"game": "checkers"}'
   - text: "Which game would you like to play?"
-    buttons:
-    - title: "Chess"
-      payload: '/inform{"game": "chess"}'
-    - title: "Checkers"
-      payload: '/inform{"game": "checkers"}'
 ```
 
-When using channel-specific responses, the value of the `channel` key should match
-the name used to provide the credentials for that channel
-in the `credentials.yml` file.
+:::note
+Make sure the value of the `channel` key matches the channel name used in your `credentials.yml` file.
```

Comment:
I am actually not sure if this is the case 🤔  I think we want them to use the `name` returned by the input channel. In the case of existing connectors, those match, but in the case of a custom one, your credentials would have path.to.class.module instead of the name of the channel.

Reply:
@erohmensing I know nothing about this, it's just that the original wording didn't make sense to me. Would you be able to suggest something that's technically correct and worded lightly?

Reply:
Make sure the value of the `channel` key matches the value returned by the `name()` method of your input channel. If you are using a built-in channel, this value will also match the channel name used in your `credentials.yml` file. 

Does that make sense? ^

Reply:
for example here https://github.com/RasaHQ/rasa/blob/master/rasa/core/channels/facebook.py#L26

Reply:
Makes sense to me, thanks :-)